src/FakeItEasy/Core/AssemblyExtensions.cs

```diff
@@ -14,16 +14,7 @@ public static bool ReferencesFakeItEasy(this Assembly assembly)
         {
             Guard.AgainstNull(assembly, nameof(assembly));
 
-#if FEATURE_REFLECTION_GETASSEMBLIES
             return assembly.GetReferencedAssemblies().Any(r => r.FullName == TypeCatalogue.FakeItEasyAssembly.FullName);
-#else
-            var fakeItEasyLibraryName = TypeCatalogue.FakeItEasyAssembly.GetName().Name;
-            var context = Microsoft.Extensions.DependencyModel.DependencyContext.Default;
-
-            var runtimeLib = context.RuntimeLibraries
-                .SingleOrDefault(library => string.Equals(library.Name, assembly.Name(), System.StringComparison.Ordinal));
-            return runtimeLib != null && runtimeLib.Dependencies.Any(dependency => string.Equals(dependency.Name, fakeItEasyLibraryName, System.StringComparison.Ordinal));

```

Comment:
Wow. @adamralph's comment from 30 minutes ago shows up above @thomaslevesque's of 11 days ago. This confuses me.

This code came with @jeremymeng's PR. He was working through several .NET Core pre-releases, so perhaps `GetReferencedAssemblies` wasn't available when he first made his changes and there was no (known) reason to go back and alter it?